File: Proyectos Kevin/Basico_C#/Actividad8/modelos/Tarea.cs
using System;

public class Tarea
{
    // Propiedades automáticas (simplificadas)
    public string Titulo { get; set; }
    public string Descripcion { get; set; }
    public bool Completada { get; private set; } // Solo modificable dentro de la clase

    // Constructor
    public Tarea(string titulo, string descripcion, bool completada = false)
    {
        Titulo = titulo;
        Descripcion = descripcion;
        Completada = completada;
    }

    // Método para cambiar el estado de la tarea a completada
    public void MarcarComoCompletada()
    {
        Completada = true;
    }

    // Método para obtener una representación de la tarea en forma de texto
    public override string ToString()
    {
        // Usa una cadena más clara para representar el estado
        string estado = Completada ? "✓ Completada" : "⏳ Pendiente";
        return $"{Titulo} - {Descripcion} [{estado}]";
    }
}

File: Proyectos Kevin/Basico_Java/src/modelos/Tarea.java
package modelos;

public class Tarea {
    String titulo;
    String descripcion;
    Boolean estado;
    
    public Tarea(String titulo, String descripcion, boolean estado){
        this.titulo = titulo;
        this.descripcion = descripcion;
        this.estado = estado;
    }
    
    public String getTitulo() {
        return titulo;
    }
    
    public String getDescripcion() {
        return descripcion;
    }
    
    public Boolean getEstado() {
        return estado;
    }
    
    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }
    
    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
    
    public void setEstado(boolean estado) {
        this.estado = estado;
    }

    // Método para obtener la representación del estado de la tarea
    public String getEstadoString() {
        return estado ? "Completada" : "Pendiente";
    }

    @Override
    public String toString() {
        return estado ? "[✅] " + titulo + " " + descripcion :
                        "[ ] " + titulo + " " + descripcion;
    }
}
